vars/utils.groovy

```diff
@@ -397,12 +397,24 @@ def perfEval_gfx908() {
             echo "#errored jobs: ${errored_conv_jobs}"
             error("Unable to eval all conv jobs")
         }
+
+        def last_gold_v = runsql("SELECT max(golden_miopen_v) from conv_golden;")
+        def next_gold_v = last_gold_v.toInteger() + 1
+        sh "./tuna/populate_golden.py --session_id ${sesh1} --golden_v ${next_gold_v} --base_golden_v ${last_gold_v}"
+        def golden_entries = runsql("SELECT count(*) from conv_golden where session= ${sesh1};")
+        def fdb_entries = runsql("SELECT count(*) from conv_golden where session= ${sesh1};")
+        if(golden_entries.toInteger() != fdb_entries.toInteger())
+        {
+            echo "#fdb jobs: ${fdb_entries}"
+            echo "#goden jobs: ${golden_entries}"
+            error("FDB entries and golden entries do not match")
+        }
```

Comment:
Does this add any additional coverage compared to the unit test ?

Reply:
likely not

Reply:
coverage is not computed from pipeline calls, only from our unit tests.